src/main/java/net/fuzzycraft/botanichorizons/patches/GregtechPatches.java

```diff
@@ -329,43 +357,32 @@ public static IRecipe addQuartzRecipes(int quartzMeta, @Nullable Object ingredie
                         'C', ingredient));
             } else {
                 // Smokey Quartz - deduplicate with Thaumic Tinkerer
-                GT_ModHandler.addAlloySmelterRecipe(new ItemStack(Items.quartz, 8), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Coal, 1), new ItemStack(ModItems.quartz, 8), 400, 16, false);
-                GT_ModHandler.addAlloySmelterRecipe(new ItemStack(Items.quartz, 8), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Charcoal, 1), new ItemStack(ModItems.quartz, 8), 400, 16, false);
-            }
+                GT_Values.RA.stdBuilder()
+                        .itemInputs(new ItemStack(Items.quartz, 8), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Coal, 1))
+                        .itemOutputs(new ItemStack(ModItems.quartz, 8))
+                        .duration(20 * SECONDS)
+                        .eut(16)
+                        .addTo(RecipeMaps.alloySmelterRecipes);
+                GT_Values.RA.stdBuilder()
+                        .itemInputs(new ItemStack(Items.quartz, 8), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Charcoal, 1))
+                        .itemOutputs(new ItemStack(ModItems.quartz, 8))
+                        .duration(20 * SECONDS)
+                        .eut(16)
+                        .addTo(RecipeMaps.alloySmelterRecipes);
+               }
             return BotaniaAPI.getLatestAddedRecipe();
         } else {
             return null;
         }
     }
 
     public static void addGlassPane(Block paneBlock, Block inputBlock) {
-        RecipeMaps.cutterRecipes.addRecipe(
-                true,
-                new ItemStack[]{new ItemStack(inputBlock, 3)},
-                new ItemStack[]{new ItemStack(paneBlock, 8)},
-                null, null,
-                new FluidStack[]{Materials.Water.getFluid(4)},
-                null,
-                100, 7, 0
-        );
-        RecipeMaps.cutterRecipes.addRecipe(
-                true,
-                new ItemStack[]{new ItemStack(inputBlock, 3)},
-                new ItemStack[]{new ItemStack(paneBlock, 8)},
-                null, null,
-                new FluidStack[]{GT_ModHandler.getDistilledWater(3)},
-                null,
-                100, 7, 0
-        );
-        RecipeMaps.cutterRecipes.addRecipe(
-                true,
-                new ItemStack[]{new ItemStack(inputBlock, 3)},
-                new ItemStack[]{new ItemStack(paneBlock, 8)},
-                null, null,
-                new FluidStack[]{Materials.Lubricant.getFluid(1)},
-                null,
-                50, 7, 0
-        );
+        GT_Values.RA.stdBuilder()
+                .itemInputs(new ItemStack(inputBlock, 3))
+                .itemOutputs(new ItemStack(paneBlock, 8))
+                .duration(2*SECONDS+10*TICKS)
+                .eut(7)
+                .addTo(cutterRecipes);
```

Comment:
Fluid requirements and speed differences have been removed, please restore them.